### DISCUSSION TITLE
Do you also have a email app?

### DISCUSSION THREAD

**Discussion by grd** [helloSystem/hello#303] [Q&A]
In the perfect presentation at FOSDEM I saw a lot of beautiful things, so I got interested. This is why I ask whether you also have a email app?

**Comment by probonopd** [comment]
Right now we don't ship one with helloSystem yet, but there is e.g., https://github.com/helloSystem/Applications/releases/tag/Thunderbird. Please let us know how well it works for you, or whether you know a better suited one.

**Reply by grd** [reply]
Thank you for your answer. Yes, I am gonna try out Thunderbird. But if it is not gonna work then I might try out to write a graphical cover for mutt.

**Comment by alphamodh0** [comment]
Maybe a good option: https://github.com/VolkMilit/CuteMail
Mail client with multiple accounts support.

No browser.

No javascript.

No calendar and this kind of things. Never understend, why using calendar in email client?

Junk mail must be moved to junk folders, without deleting, user must dicede what (s)he want to do with them.

Clear and understandable interface - no useless buttons and settings. (hello, Thunderbird!)

Dependencies

Qt5

Compiller with C++11 support

libetpan

**Reply by probonopd** [reply]
Looks interesting, but cannot get it to compile: https://github.com/VolkMilit/CuteMail/issues/35

**Comment by alphamodh0** [comment]
Sorry, but Thunderbid is a headache to me.

**Comment by probonopd** [comment]
How about [Trojitá](http://trojita.flaska.net/)? It can do IMAP and SMTP. `sudo pkg install -y trojita && trojita`

If it passes your test, then maybe we should add it to the curated default applications.

**Reply by alphamodh0** [reply]
Wow, Trojitá should definitely be the email client for helloSystem. It is more than approved. Thanks 👍🏻.

**Reply by probonopd** [reply]
Adding to the next experimental build.

**Comment by alphamodh0** [comment]
![trojita](https://user-images.githubusercontent.com/90866187/147880208-fe5bc374-9c00-4550-a667-842f39f3ff19.png)
![trojita2](https://user-images.githubusercontent.com/90866187/147880230-f2261867-6664-4b1e-a198-98d4e79dc54c.png)

**Reply by probonopd** [reply]
Very nice. The only thing that is irritating me - what is that WLAN icon doing in the mail application?

**Reply by alphamodh0** [reply]
> Very nice. The only thing that is irritating me - what is that WLAN icon doing in the mail application?

Is a kind of net mode.

![20220102_102901.jpg](https://user-images.githubusercontent.com/90866187/147880616-d9c6f911-a660-405b-acf8-c1fd7bbdf215.jpg)

**Reply by probonopd** [reply]
I wish they were using a different icon, not the WLAN one. Oh well.

**Comment by grd** [comment]
Yes, Trojita works! Brilliant!

**Comment by grd** [comment]
After using Trojita for a while now, I have to say that there are two flaws.

1. Filtering messages is not there. This is important when you use a large amount of emails or folders.
2. The last release of Trojita has been 5 years ago (according to Wikipedia). Which shows that this project appears abandoned.

**Reply by probonopd** [reply]
There is still some activity going on: https://github.com/KDE/trojita/commits/master

I'd take it as a positive that the software is mature and no longer in a constant state of change. I like "complete" software :)

> Filtering messages is not there. This is important when you use a large amount of emails or folders.

https://github.com/KDE/trojita/search?q=filtering mentions "Widget containing a list of messages along with quick filtering toolbar". Not sure whether this is what you were looking for.
